--- a/papilo/CMakeLists.txt
+++ b/papilo/CMakeLists.txt
@@ -214,7 +214,6 @@ write_basic_package_version_file(
 )
 
 if(MSVC)
-   target_compile_definitions(papilo INTERFACE BOOST_ALL_NO_LIB)
    target_compile_options(papilo INTERFACE /bigobj)
    target_compile_options(papilo INTERFACE ${TBB_CXX_STD_FLAG})
 else()
--- a/soplex/CMakeLists.txt
+++ b/soplex/CMakeLists.txt
@@ -211,7 +211,6 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
     add_definitions(/wd4100)
     add_definitions(/wd4661)
     add_definitions(-D_CRT_SECURE_NO_WARNINGS)
-    add_definitions(-DBOOST_ALL_NO_LIB)
     # set MT or MTd flag when using the static VS runtime libraries
     if(MT)
         set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /MT")
